src/Template/Element/deprecated_panel.ctp

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         DebugKit 2.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
+/**
+ * @var \DebugKit\View\AjaxView $this
+ * @var array $paths
+ * @var array $app
+ * @var array $cake
+ * @var array $plugins
+ * @var array $vendor
+ * @var array $other
+ */
+
+// Backwards compat for old DebugKit data.
+if (!isset($cake) && isset($core)) {
+    $cake = $core;
+}
+?>
+
+<h4><?= __d('debug_kit', 'Deprecated methods paths') ?></h4>
+<?= $this->Toolbar->makeNeatArray(compact('app', 'cake', 'plugins', 'vendor', 'other')) ?>
```

Comment:
It might be easier to read if this information was presented as a table for each package/app. Right now you'd need a lot of clicking to quickly scan your deprecations.

Reply:
a bit changed

Reply:
what's the decision on this?

Reply:
nothing yet,
I no have idea about this.